docs/source/release/v6.4.0/framework.rst

```diff
@@ -21,6 +21,8 @@ Data Objects
 Python
 ------
 
+- Added possibility to forward log messages to Python, see `mantid.utils.logging.log_to_python`.
```

Comment:
Please make this a link to `dev-docs/source/Logging.rst`

Reply:
How do you link from the user to the dev docs? Is there a nicer way than using URLs to the published online documentation?

Reply:
It should be
```
:ref:`mantid.utils.logging.log_to_python`
```

Reply:
Ah, ok.
But I was also asking about your other comment where I need to link to  a file / section.